tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+from tests import (  # noqa
```

Comment:
Fairly sure I didn't, but I'll have a poke in a sec. It worked in python2 anyway, so the only thing that springs to mind is relative import issues...